rdmorganiser/questions/rdmo.xml

```diff
@@ -118,12 +118,12 @@
 		<is_collection>True</is_collection>
 		<order>0</order>
 		<help lang="en">The list of disciplines follows the &lt;a href=&quot;http://www.dfg.de/en/dfg_profile/statutory_bodies/review_boards/subject_areas/index.jsp&quot; target=_blank&quot;&gt;subject classification of the DFG (German Research Foundation)&lt;/a&gt;.</help>
```

Comment:
`[...]/index.jsp&quot; target=_blank&quot;&gt;` Is there a `&quot;` missing before `_blank`?

Reply:
yes. Introduced from upstream. I opened an issue https://github.com/rdmorganiser/rdmo-catalog/issues/15

Reply:
see 5ae5cb6fb7ee487d7aec2292544182f64e188226   and 27f82be6f1007326f7795c469641ee86bd7f54bc